recipes/sphinx-gallery/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "sphinx-gallery" %}
+{% set version = "0.1.7" %}
+{% set sha256 = "594da5b4ed08e6531798c735608f4fe65a8bd394ea3b7b9194d9e659e48a4ea0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - nose
+    - matplotlib
+    - pillow
+    - sphinx
```

Comment:
I think you're right, I'll modify the recipe in this direction.

Here is the `requirements.txt`:

    - setuptools
    - matplotlib
    - pillow
    - sphinx
    - nose

`install_requires` is:

    # XXX : don't force requirements in setup.py as it tends to break
    #       people's setups
    install_requires = []

    if os.environ.get('READTHEDOCS', None) == 'True':
        # So that we can build documentation using seaborn
        install_requires = ['seaborn']

`setup_requires` is:

    setup_requires=['nose>=1.0']